reactLab/src/Modal.js

```diff
@@ -0,0 +1,36 @@
+import './Modal.css';
+
+function Modal(props){
```

Comment:
I'd extract out from here a component that is completely reusable (displays its children and has ok and cancel buttons) and then use that component (which I'd call Modal) to create your own DeleteConfirmModal.

Modal could then be used other times within your project (perhaps for later labs), as well as for other projects.